pkg/filterengine/filters/ignore-namespace-filter/ignore_namespace_checker.go

```diff
@@ -0,0 +1,63 @@
+package filters
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracloudio/botkube/pkg/config"
+	"github.com/infracloudio/botkube/pkg/events"
+	"github.com/infracloudio/botkube/pkg/filterengine"
+	log "github.com/infracloudio/botkube/pkg/logging"
+)
+
+// IgnoreNamespaceChecker ignore events from blocklisted namespaces
+type IgnoreNamespaceChecker struct {
```

Comment:
Can we name this filter to something like - NamespaceFilter or NamespaceChecker. IgnoreNamespaceChecker sounds little odd.

Reply:
sure, will change it to NamespaceChecker

Reply:
resolved it